src/test/java/org/minions/devfund/alejandro/tests/SquarelotronTest.java

```diff
@@ -0,0 +1,287 @@
+package org.minions.devfund.alejandro.tests;
+
+import org.junit.Test;
+import org.minions.devfund.alejandro.Squarelotron;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.Before;
+
+import java.util.Arrays;
+
+/**
+ * Class that will test the class Squarelotron.
+ */
+public class SquarelotronTest {
+
+    private Squarelotron squarelotron;
+    private Squarelotron actualSquarelotron;
+
+
+
+    /**
+     * This method will initialize the Squarelotron class.
+     */
+    @Before
+    public void setUp() {
+        final int size = 5;
+        squarelotron =  new Squarelotron(size);
```

Comment:
The setup method is created when it is reused the attributes of class in all the tests
So in this case you are not reusing squarelotron instance with size 5 in all your test.
Please fix this problem.